app/src/main/java/co/condorlabs/customcomponents/customtextview/readme.md

```diff
@@ -45,6 +45,7 @@
 |  SUBTITLE_TYPE | 2 |
 |  SECTION_TITLE_TYPE | 3 |
 |  LINK_TYPE | 4 |
+|  h2_TITLE_TYPE | 5 |
```

Comment:
in capital letters?

Reply:
are constants, so the documentation goes with the same type

Reply:
done